src/components/dashjs-statistics/dashjs-statistics.tsx

```diff
@@ -1,7 +1,7 @@
-import { Component, Host, h, Prop, Watch, Element, State, Listen } from '@stencil/core';
+import { Component, Host, h, Element, State, Listen } from '@stencil/core';
 import * as chartjs from 'chart.js';
 const { Chart } = chartjs.default.Chart;
-import { calculateHTTPMetrics, chartDataset, chartYAxisOptions } from '../../utils/metrics';
+import { chartDataset, chartYAxisOptions } from '../../utils/utils';
```

Comment:
please use the `utils/metrics` the utils file will get enormous otherwise

Reply:
done